src/include/cdb/cdbdtxcontextinfo.h

```diff
@@ -15,7 +15,7 @@
 #define CDBDTXCONTEXTINFO_H
 #include "utils/tqual.h"
 
-#define DtxContextInfo_StaticInit {0,InvalidDistributedTransactionId,0,false,false,DistributedSnapshot_StaticInit,0,0}
+#define DtxContextInfo_StaticInit {0,0,false,false,DistributedSnapshot_StaticInit,0,0,InvalidDistributedTransactionId}
```

Comment:
```suggestion
#define DtxContextInfo_StaticInit                                              \
  {                                                                            \
    .distributedTimeStamp = 0,                                                 \
    .distributedXid = InvalidDistributedTransactionId,                         \
    .haveDistributedSnapshot = false, .cursorContext = false,                  \
    .distributedSnapshot = DistributedSnapshot_StaticInit,                     \
    .distributedTxnOptions = 0, .segmateSync = 0, .nestingLevel = 0,           \
  }
  
```

Here's a 90-second sketch of that. Things that may or may not be right in my patch:

1. I'm using multiple lines and aligning the RHS at 80 columns. Can't find what the style guide says about either (multi-line and RHS-alignment)
2. This leaves curcid implicitly zero'd. Do we want to be explicit about it?